interfaces/Glitter/templates/static/javascripts/glitter.main.js

```diff
@@ -388,6 +414,23 @@ function ViewModel() {
             self.isRestarting(1)
         }).always(self.setNextUpdate);
 
+        // this is a test of getting the entire queue
```

Comment:
yeah.  ok so i will be removing all the crap code ;)  and generating a new commit.  busy day today so this will likely come tonight/tomorrow.